src/components/Claim/index.tsx

```diff
@@ -1,36 +1,247 @@
+    setAmountError(undefined)
+
+    setIsMaxAmountSelected(true)
+  }
 
   return (
-    <MediumPaper>
-      <ProgressBar value={progress} />
-      {steps[step]}
-    </MediumPaper>
+    <PaperContainer>
+      <Grid container spacing={3} direction="row" justifyContent="space-evenly">
+        <Grid item container xs={12} spacing={3}>
+          <Grid item xs={12} sm={6}>
+            <ClaimCard
+              variant="claimable"
+              isGuardian={!!ecosystemVesting}
+              totalAmount={total.claimable}
+              ecosystemAmount={ecosystem.claimable}
+              decimals={decimals}
+            />
+          </Grid>
+          <Grid item xs={12} sm={6}>
+            <ClaimCard
+              variant="vesting"
+              isGuardian={!!ecosystemVesting}
+              totalAmount={total.inVesting}
+              ecosystemAmount={ecosystem.inVesting}
+              decimals={decimals}
+            />
+          </Grid>
+        </Grid>
+        <Grid item pt={2} xs={12}>
+          <Stack spacing={3}>
+            <InfoAlert>
+              <Typography variant="body2" color="text.secondary">
+                Total awarded allocation is{' '}
+                <Typography component="span" variant="inherit" color="text.primary">
+                  {formatAmount(formatEther(total.allocation), 2)} SAFE
+                </Typography>
+              </Typography>
+            </InfoAlert>
+
+            <Paper
+              sx={{
+                p: 3,
+                backgroundColor: ({ palette }) => palette.background.default,
+                color: ({ palette }) => palette.text.primary,
+                position: 'relative',
+                height: '100%',
+                display: 'flex',
+                flexDirection: 'column',
+                gap: 1,
+              }}
+            >
+              <Stack direction="row" spacing={2} alignItems="center">
+                <SvgIcon fontSize="large" component={TitleStar} />
+                <Box>
+                  <Typography variant="subtitle1" fontWeight={700}>
+                    Claim your tokens as rewards
+                  </Typography>
+                  <Typography variant="body2">You get more tokens if you are active in activity program.</Typography>
+                </Box>
+              </Stack>
+            </Paper>
+
+            <Divider />
+
+            <Box>
+              <Typography variant="h6" mt={2} fontWeight={700}>
+                How much do you want to claim?
+              </Typography>
+              <Typography color="text.secondary" variant="body1">
+                Select all tokens or define a custom amount.
+              </Typography>
+            </Box>
+
+            {canRedeemSep5 && (
+              <Grid item xs={12}>
+                <Sep5InfoBox />
+              </Grid>
+            )}
+
+            <Grid item container gap={3} flexWrap="nowrap" xs={12} mb={1}>
+              <Grid item xs={4}>
```

Comment:
The TextField and Claim button only take up half the size with xs 4 and 2 while the delegation components underneath take up the full space.
This looks a bit misaligned:
![Screenshot 2024-03-25 at 07 24 20](https://github.com/safe-global/safe-dao-governance-app/assets/2670790/9f68c9ea-afea-4783-8b67-8c9dfe3d3b1c)

In the designs the claiming area is a bit wider and the delegates do not take up 100% width.
Probably it would be best to align them by taking 9/12 breakpoints for instance.